selfdrive/car/volkswagen/carcontroller.py

```diff
@@ -38,36 +39,31 @@ def update(self, CC, CS, ext_bus, now_nanos):
       #   * Don't send > 3.00 Newton-meters torque
       #   * Don't send the same torque for > 6 seconds
       #   * Don't send uninterrupted steering for > 360 seconds
-      # One frame of HCA disabled is enough to reset the timer, without zeroing the
-      # torque value. Do that anytime we happen to have 0 torque, or failing that,
-      # when exceeding ~1/3 the 360 second timer.
+      # MQB racks reset the uninterrupted steering timer after a single frame
+      # of HCA disabled; this is done whenever output happens to be zero.
```

Comment:
This sounds very similar to the workarounds for Toyota and Hyundai. Can you not also set the bit to 0 while torque != 0? Does it fault? Wind down torque and take a while to back up? Any other problems?

Reply:
I think I can get away with that on MQB, or at least that's what my three-year-old code/comments say. But I decided to discard that case because:

1) It adds complexity for a case I don't think we ever hit in the real world; we get resets at zero very frequently
2) We gain validation for that frequent zero assumption with the new Event, which we didn't have before
3) You've recently added LKA bit enabled validation in Panda, and I'd like to include VW in that
4) The upcoming PR to mitigate the PQ/MLB 6 minute timeout definitely needs to hold LKA *and* torque at zero